pandas/tests/test_common.py

```diff
@@ -229,3 +230,12 @@ def test_temp_setattr(with_exception):
                 raise ValueError("Inside exception raised")
         raise ValueError("Outside exception raised")
     assert ser.name == "first"
+
+
+def test_str_size():
+    # GH#21758
+    str = "a"
+    expected = sys.getsizeof(str)
+    import pandas as pd
```

Comment:
the pandas import won't do anything here as pandas is already imported.

the only way this could work is via a subprocess tests i think (look at test_downstream.py)

Reply:
Or I think you can use importlib.reload https://docs.python.org/3/library/importlib.html#importlib.reload

Reply:
@mroeschke I don't think reloading will work as you need to actually "clean" the module. 

@jreback were you mentioning this? https://github.com/pandas-dev/pandas/blob/fe9e5d023e20304ad1bdfa1da53f3af452c72a00/pandas/tests/test_downstream.py#L136-L138

I believe that's the only proper way of doing it without breaking anything else. 

Opening a Python process with `spawn` changes the mechanism for the whole `multiprocess`, if you have other ideas, let me know. Otherwise, I'd be interested in picking this back up and submitting a PR